dbms/src/Storages/Page/WriteBatch.h

```diff
@@ -86,6 +86,7 @@ class WriteBatch : private boost::noncopyable
         : writes(std::move(rhs.writes))
         , sequence(rhs.sequence)
         , namespace_id(rhs.namespace_id)
+        , total_data_size(rhs.total_data_size)
```

Comment:
The `swap` method did not update the field `namespace_id`.
https://github.com/pingcap/tiflash/blob/28a209a39c947b590b62da9ff249605a35b2da1f/dbms/src/Storages/Page/WriteBatch.h#L190-L195

Maybe we should move the `WriteBatch(WriteBatch && rhs)` and `swap` next to the private variables definitions so that we can check whether these methods miss setting variables

Reply:
Fixed.